modules/FindMagnumIntegration.cmake

```diff
@@ -190,6 +191,14 @@ foreach(_component ${MagnumIntegration_FIND_COMPONENTS})
 
             set(_MAGNUMINTEGRATION_${_COMPONENT}_INCLUDE_PATH_NAMES MotionState.h)
 
+        # ImGui integration library
+        elseif(_component STREQUAL ImGui)
+            find_package(ImGui REQUIRED)
```

Comment:
Not sure about the `REQUIRED` here -- if `find_package(MagnumIntegration OPTIONAL)` would be used, this will break. (But OTOH I also don't know what's the *proper* way of doing this, I'm only saying `find_package(Name)` everywhere.)

Reply:
Ah, right, now I remember why we had a discussion about this once :D

Reply:
[Documentation](https://cmake.org/cmake/help/v3.0/command/find_package.html) says "REQUIRED stops processing on error message", hence manually outputting an error message would make it stop if `find_package(MagnumIntegration REQUIRED ImGui)` whereas OPTIONAL would print the error message but continue processing...?

Reply:
OTOH, the error message will be output by `find_package(ImGui)` already... right? So it will stop processing there, I'd expect...

Reply:
Not sure, probably not. We need to revisit this later, maybe we need to add `REQUIRED` based on value of [MagnumIntegration_FIND_REQUIRED_ImGui](https://cmake.org/cmake/help/latest/command/find_package.html#id6) ... but that then needs to be done for *everything*.

Reply:
Can you remove the `REQUIRED`, then?